# NOTE: You have to change this node name if you change the node name in the launchfile.
face_detection:
  ros__parameters:
    # The name of the face detection model checkpoint to use
    face_model_name: haarcascade_frontalface_alt2.xml
    # The URL to download the face detection model checkpoint from if it is not already downloaded
    face_model_base_url: "https://raw.githubusercontent.com/opencv/opencv/master/data/haarcascades/"
    # The name of the facial landmark detection model checkpoint to use
    landmark_model_name: lbfmodel.yaml
    # The URL to download the facial landmark detection model checkpoint from if it is not already downloaded
    landmark_model_base_url: "https://raw.githubusercontent.com/kurnianggoro/GSOC2017/master/data/"


  